### PR TITLE
Dynamic imports

### PR DESCRIPTION
**Pull request by lhmisho**

### CODE REVIEW

**Comment by jamal-pb95** [comment]
Sorry, Brother @lhmisho you have raised a PR #27 where the files belong `The old "var"` contents. but the PR title is `Dynamic imports`. hope you understand.

**Comment by lhmisho** [comment]
Dear Jamal Bhaii,
As your instruction i just separated my master into two different branch
and made PR separately.
But there is also some other issues. I am really confused to what to do.
Please help me.
Thank you.

**Comment by raikusy** [comment]
You have translated the wrong file. Please check your title and contents before creating a PR. Thank you.

**Comment by raikusy** [comment]
@lhmisho check the difference here - https://github.com/javascript-tutorial/bn.javascript.info/pull/27/files

**Comment by raikusy** [comment]
@lhmisho আপনাকে এখানে /done লিখে কমেন্ট করতে হবে। তাহলে আমরা আবার রিভিউ করতে পারবো।
ধন্যবাদ।

File: 1-js/13-modules/03-modules-dynamic-imports/article.md [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
 
-First, we can't dynamically generate any parameters of `import`.
+প্রথমত, `import` এর কোন প্যারামিটার ডাইনামিক ভাবে আমরা জেনারেট করতে পারি না। 
 
-The module path must be a primitive string, can't be a function call. This won't work:
+মডিউলের পাথ অবশ্যই প্রিমিটিভ স্ট্রিং হতে হবে, ফাংশন কল হওয়া যাবে না। এটি কাজ করবে নাঃ 
 
 ```js
-import ... from *!*getModuleName()*/!*; // Error, only from "string" is allowed
+import ... from *!*getModuleName()*/!*; এরর, শুধুমাত্র "string" প্রযোজ্য
```
Comment (raikusy) [inline 1-js/13-modules/03-modules-dynamic-imports/article.md]:
```suggestion
import ... from *!*getModuleName()*/!*; // এরর, শুধুমাত্র "string" প্রযোজ্য
```

**Comment by lhmisho** [comment]
It's always a great importunity to be a contributor of something which is
provably  going to helps our community.
And thanks Dear @Md. Jamal Uddin bhaii for your feedback.

**Pull request merged**